smartdashboard/Experiment_Overview.py

```diff
@@ -64,9 +67,10 @@ def build_app(manifest_path: str) -> None:
         )
 
         while True:
+            if manifest_reader.has_changed:
+                st.rerun()
             for v in to_update:
                 v.update()
```

Comment:
I was thinking about what might happen if the manifest changes a few times (app re-runs immediately with no sleep, which might be undesirable) and noticed that `v.update` is called for all views, regardless of visibility.

Considering the way streamlit re-runs when we navigate around, could we get a performance boost by avoiding extraneous updates? I wonder if we should add a ticket for adding a similar concept to the views 

for example: `for v in [x for x in to_update if x.visible]: v.update()`

Reply:
Love it! I'll work on a ticket for it